src/Exception/TestException.php

```diff
@@ -10,7 +10,7 @@
 /**
  * 
  */
-abstract class ExceptionAbstract extends \Exception
+abstract class TestException extends \Exception
```

Comment:
Yes. I'm not quite sure on making this abstract. WDYT.
I think its's better to throw a TestException in userland as throwing a bad or too general exception